lib/ansible/modules/cloud/google/gcp_bigquery_table_info.py

```diff
@@ -29,9 +29,11 @@
 
 DOCUMENTATION = '''
 ---
-module: gcp_bigquery_table_facts
+module: gcp_bigquery_table_info
 description:
 - Gather facts for GCP Table
+- This module was previously called gcp_bigquery_table_facts before Ansible 2.9. The
```

Comment:
We don't need both "previously" and "before Ansible 2.9". How about using the pattern "Before Ansible 2.9, this module was called {{ old_name }}."?

Reply:
Alternatively, there's `This module was called C({{ old_name }}) before Ansible 2.9.`, which I've used in a lot of other renames :)